bonsai/Modules/Onboarding/Onboarding.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Onboarding.swift
+//  bonsai
+//
+//  Created by antuan.khoanh on 06/08/2022.
+//
+
+import SwiftUI
+
+struct Onboarding: View {
+    
+    @State private var selection = 0
+    
+    private var onboardingImages = [
+        "bonsai_1",
+        "bonsai_2",
+        "bonsai_3"
+    ]
+    
+    var body: some View {
+        ZStack {
+            VStack {
+                TabView(selection: $selection) {
+                    ForEach(0..<3) { index in
```

Comment:
why not use `0..<onboardingImages.count` instead of 0..<3

Reply:
or maybe ForEach(onboardingImages) could be even better? Not sure if it will work tho

Reply:
updated+